src/Utility.js

```diff
@@ -91,17 +93,18 @@ export function sizeof (value) {
 }
 
 /**
- * @param {any} value
+ * @template T
+ * @param {T} value
  * @param {any[]} array
- * @return {any}
+ * @return {T}
  */
 export function append (value, array) {
 	return array.push(value), value
 }
 
 /**
  * @param {string[]} array
- * @param {function} callback
+ * @param {ArrayMapCallback} callback
```

Comment:
It's mostly to differentiate from Middleware which looks _super similar_ and middleware is sometimes passed around as the variable `callback` 🙄

I agree that the type is not as generic as it could be, but it's actually only used **twice** in the whole codebase and my preference would be to *not* have `combine` at all - use `[].map().join()`. The author likes to have a lot of aliases though, and I don't think it's worth trying to convince them to change that